lib/ansible/modules/cloud/amazon/aws_ssm_send_command.py

```diff
@@ -0,0 +1,212 @@
+options:
+  name:
+    description:
+      - This should match the name of the SSM document to be invoked.
+    type: str
+    required: true
+  comment:
+    description:
+      - A comment about this particular invocation.
+    required: false
+    type: str
+  instance_ids:
+    description:
+      - A list of instance IDs for the instances you wish to run this command
+        document against.
+    required: true
+    type: list
+  wait:
+    description:
+      - Whether to wait for the function results or not. If I(wait) is false,
+        the task will not return any results. To wait for the command to
+        complete, set C(wait=true) and the result will be available in the
+        I(output) key.
+    required: false
+    type: bool
+    default: true
+  parameters:
+    description:
+      - A dictionary to be provided as the parameters for the SSM document
+        you're invoking.
+    required: false
+    default: {}
+extends_documentation_fragment:
+    - aws
+    - ec2
+'''
+
+EXAMPLES = '''
+- aws_ssm_send_command:
+    name: AWS-UpdateSSMAgent
+    comment: "SSM agent update check"
+    instance_ids:
+      - i-123987193812
+      - i-289189288278
+    parameters:
+      version: latest
+      allowDowngrade: 'false'
+    wait: true
+  register: response
+'''
+
+RETURN = '''
+output:
+    description: If wait=true, will return the output of the executed command. Sample truncated for brevity.
+    returned: success
+    type: str
+    sample: "Updating amazon-ssm-agent from 2.3.539.0 to latest\nSuccessfully downloaded https://s3.us-west-2.amazonaws.com/amazon-ssm-us-west-2/ssm-agent..."
+status:
+    description: Status of the run command.
+    returned: success
+    type: str
+    sample: Success
+'''
+
+from ansible.module_utils.aws.core import AnsibleAWSModule
+from ansible.module_utils.ec2 import boto3_conn, get_aws_connection_info, AWSRetry
+import traceback
+from time import sleep
+
+try:
+    import botocore
+except ImportError:
+    pass  # will be captured by imported HAS_BOTO3
+
+
+@AWSRetry.backoff(tries=5, delay=5, backoff=2.0)
+def ssm_send_command(conn, **kwargs):
+    return conn.send_command(**kwargs)
+
+
+@AWSRetry.backoff(tries=5, delay=5, backoff=2.0)
+def ssm_list_command_invocations(conn, **kwargs):
+    return conn.list_command_invocations(**kwargs)
+
+
+def main():
+    argument_spec = dict(
+        name=dict(required=True),
+        wait=dict(default=True, type='bool'),
+        comment=dict(),
+        instance_ids=dict(required=True, type='list'),
+        parameters=dict(default={}, type='dict')
+    )
+    module = AnsibleAWSModule(
+        argument_spec=argument_spec,
+        supports_check_mode=False
+    )
+
+    # Needs to be an existing SSM document name
+    document_name = module.params.get('name')
+    comment = module.params.get('comment')
+    await_return = module.params.get('wait')
+    instance_ids = module.params.get('instance_ids')
+    parameters = module.params.get('parameters')
```

Comment:
I would like to see the other configurable options here as well, see: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ssm.html#SSM.Client.send_command
Relatively small thing to add which will make this module way more dynamic.